src/test/java/application/rest/v1/ConfigMapEndpointTest.java

```diff
@@ -0,0 +1,349 @@
+            "                  \\\"menu-item\\\": \\\"true\\\"\\n  }\\n]\\n\"\n" + 
+            "    }," + 
+            "    \"kind\": \"ConfigMap\"," + 
+            "    \"metadata\": {" + 
+            "        \"name\": \"kappnav-test-cell1-New\"," + 
+            "        \"namespace\": \"stock-trader\"," + 
+            "        \"resourceVersion\": \"1221195\"," + 
+            "        \"selfLink\": \"/api/v1/namespaces/stock-trader/configmap/kappnav-test-cell1\"," + 
+            "        \"uid\": \"7a08855f-71c7-4bf6-8cee-57c1b72175fb\"" + 
+            "    }," + 
+            "    \"type\": \"Opaque\"" + 
+            "}" + 
+            "";
+    
+    JsonObject jsonObject = new JsonParser().parse(test1).getAsJsonObject();
+
+    /**
+     * @throws java.lang.Exception
+     */
+    @Before
+    public void setUp() throws Exception {
+        cmep.setCoreV1ApiForInternal(cv1a);
+        cmep.setApiClientForInternal(ac);
+        json.setGson(gson);
+    }
+
+    /**
+     * @throws java.lang.Exception
+     */
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    @Test
+    public void createConfigMap_succeeds() throws Exception {
+        mock.checking(new Expectations() {
+            {
+                oneOf(cv1a).setApiClient(with(any(ApiClient.class)));
+                oneOf(ac).getJSON();
+                will(returnValue(json));
+                oneOf(cv1a).createNamespacedConfigMap(with(any(String.class)), with(any(V1ConfigMap.class)), with(aNull(String.class)));
+            } 
+        });
+
+        try {
+            response = cmep.createConfigMap(test1, "stock-trader");
+            int rc = response.getStatus();
+            assertEquals("Test createConfigMap_succeeds FAILED", 200, rc);
+        } catch (Exception e) {
+            fail("Test createConfigMap_succeeds failed with exception " + e.getMessage());
+        }
+    }
+    
+    @Test
+    public void createConfigMapThrowJsonSyntaxException_error() throws Exception {
+        mock.checking(new Expectations() {
+            {
+                oneOf(cv1a).setApiClient(with(any(ApiClient.class)));
+                oneOf(ac).getJSON();
+                will(returnValue(json));
+                oneOf(cv1a).createNamespacedConfigMap(with(any(String.class)), with(any(V1ConfigMap.class)), with(aNull(String.class)));
+                will(throwException(new JsonSyntaxException("Injection to throw JsonSyntaxException.")));
+            } 
+        });
+
+        try {
+            response = cmep.createConfigMap(test1, "stock-trader");
+            int rc = response.getStatus();
+            assertEquals("Test createConfigMapThrowJsonSyntaxException_error FAILED", 400, rc);
+        } catch (Exception e) {
+            fail("Test createConfigMapThrowJsonSyntaxException_error failed with exception " + e.getMessage());
+        }
+    }
+    
+    @Test
+    public void createConfigMapThrowApiException_error() throws Exception {
```

Comment:
Because the catch block in the target method handles both exceptions identically we don't need this error test case - it covers the exact path as the first error test case.